src/main/kotlin/lotto/domain/Rank.kt

```diff
@@ -1,27 +1,16 @@
 package lotto.domain
 
-enum class Rank(val prize: Double) {
-    FIRST(Rank.PRIZE_1ST),
-    SECOND(Rank.PRIZE_2ND),
-    THIRD(Rank.PRIZE_3RD),
-    FOURTH(Rank.PRIZE_4TH),
-    LOSE(Rank.PRIZE_LOSER);
+enum class Rank(val countOfMatch: Int, val matchedBonus: Boolean, val prize: Double) {
+    FIRST(6, false, 2_000_000_000.0),
+    SECOND(5, true, 30_000_000.0),
+    THIRD(5, false, 1_500_000.0),
+    FOURTH(4, false, 50_000.0),
+    FIFTH(3, false, 5_000.0),
+    LOSE(0, false, 0.0);
```

Comment:
`SECOND`만 예외적으로 `matchedBonus`가 `true`이고 다른경우에는 `false`일것 같아보입니다.
이러한 상황에서는 `namedArgument`와 기본값 설정을 통해 다음과 같이 표현할 수 있습니다.

그리고 enum을 선언할때 마지막에 콤마를 붙이고 그 아래에 세미콜론을 둔다면 새로운 enum을 선언할때 코드리뷰시 나타나는 code change를 최소화 할 수 있습니다.
```suggestion
enum class Rank(val countOfMatch: Int, val matchedBonus: Boolean = false, val prize: Double) {
    FIRST(countOfMatch = 6, prize = 2_000_000_000.0),
    SECOND(countOfMatch = 5, prize = 30_000_000.0, matchedBonus = true),
    THIRD(countOfMatch = 5, prize = 1_500_000.0),
    FOURTH(countOfMatch = 4, prize = 50_000.0),
    FIFTH(countOfMatch = 3, prize = 5_000.0),
    LOSE(countOfMatch = 0, prize = 0.0),
    ;
```